### PR TITLE
added support for nft transfer from contract to user for MultiSig and governance

### PR DESCRIPTION
**Pull request by shivamgarglab49**
Signed-off-by: shivam_garg <shivam.garg@iongroup.com>

### CODE REVIEW

File: contracts/common/TokenOperations.sol [hunk 1]
```diff
@@ -28,28 +42,22 @@ contract TokenOperations {
     }
 
     function _transferToken(
+        IHederaService _hederaService,
         address _token,
         address _sender,
         address _receiver,
         uint256 _amount
```
Comment (amandeeprehal-lab49) [inline contracts/common/TokenOperations.sol:49]: You can say amountOrId like at other places.

File: integrationTest/governance/governorTransferToken.ts [hunk 2]
```diff
@@ -58,7 +58,7 @@ const fungibleTokenFlow = async () => {
     txnFeePayerClient,
```
Comment (amandeeprehal-lab49) [inline integrationTest/governance/governorTransferToken.ts:58]: Pls rename fungibleTokenFlow to tokenTransferWithFungibleTokenAsGODToken

File: integrationTest/governance/governorTransferToken.ts [hunk 3]
```diff
@@ -140,9 +166,9 @@ const nonFungibleTokenFlow = async () => {
   await governor.initialize(
     nftHolder,
     txnFeePayerClient,
-    500,
+    1,
```
Comment (amandeeprehal-lab49) [inline integrationTest/governance/governorTransferToken.ts:169]: nonFungibleTokenFlow pls rename this as well.

File: contracts/common/TokenOperations.sol (hunk 1 above)
Comment (shivamgarglab49) [reply contracts/common/TokenOperations.sol:49]: done

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Pull request merged**